cmake/Findquatlib.cmake

```diff
@@ -65,19 +65,35 @@ else()
 		/usr/local)
 
 	# Look for the library.
-	find_library(QUATLIB_LIBRARY
+	find_library(QUATLIB_LIBRARY_RELEASE
 		NAMES
 		quat.lib
 		libquat.a
-		HINTS
+		PATH_SUFFIXES
+		ReleaseAcademicEdition
+		Release
```

Comment:
You might, however, consider adding RelWithDebInfo and MinSizeRel after Release.